server/src/main/java/org/opensearch/rest/action/admin/indices/RestViewAction.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.rest.action.admin.indices;
```

Comment:
@peternied Can you add in the description the APIs this PR adds, describe basic use cases and what parameters they take?

Reply:
Good callout, I'll create a draft of the documentation which should help clear those gaps - then we can figure out what goes in / out this change.